pkg/cli/cli.go

```diff
@@ -132,10 +132,12 @@ func (c CLI) printReports(reports []reporter.Report) error {
 		return c.printGroupTriple(reports)
 	}
 
-	err := c.Reporter.Print(reports)
-	if err != nil {
-		fmt.Println("failed to report:", err)
-		errorFound = true
+	for _, reporterObj := range c.Reporters {
```

Comment:
I'm curious about what would happen if somehow ask for `standard,json,standard,json`

Reply:
It would do the operations twice, I thought of it but wanted to know how you would want it solved? I could use a map before creating chosenReporters to ensure we onlt have unique ones.

Reply:
You could use slices.Sort then slice.Compact

But then the order won't be respected, but it could be enough

Otherwise, yes, a map[string]bool to list the one already found coumdibe used to skip the one already provided and respect the order anyway (or report an error, as it's strange to ask multiple times the same format)

Reply:
Decided to just do sort into compact since that seemed cleaner and there isn't really a need to maintain the order for this.